app/src/lang/translations/en-US.yaml

```
@@ -119,6 +119,13 @@ hook: Hook
 fill_width: Fill Width
 field_name_translations: Field Name Translations
 enter_password_to_enable_tfa: Enter your password to enable Two-Factor Authentication
+oauth_2fa_setup_notice:
```

Wonder if it should be `sso` instead, and also the tfa-2fa inconsistency with existing translations. 🤔 

```suggestion
sso_tfa_setup_notice:
```